weapon-config.inc

```diff
@@ -2635,13 +2642,12 @@ public OnPlayerDeath(playerid, killerid, reason)
 
 		new animlib[32], animname[32], anim_lock, respawn_time;
 
-		OnPlayerPrepareDeath(playerid, animlib, animname, anim_lock, respawn_time);
+		CallLocalFunction("OnPlayerPrepareDeath", "issii", playerid, animlib, animname, anim_lock, respawn_time);
```

Comment:
Does this need a length parameter after each string parameter?

Reply:
No, because the size of the array is has a declared size
```
forward OnPlayerPrepareDeath(playerid, WC_CONST animlib[32], WC_CONST animname[32], &anim_lock, &respawn_time);
```

Reply:
ah, of course!